library/core/src/slice/mod.rs

```diff
@@ -3785,10 +3786,12 @@ impl<T> [T] {
     /// [`copy_from_slice`]: slice::copy_from_slice
     /// [`split_at_mut`]: slice::split_at_mut
     #[stable(feature = "clone_from_slice", since = "1.7.0")]
+    #[rustc_const_unstable(feature = "const_clone", issue = "142757")]
+    #[rustc_allow_const_fn_unstable(const_precise_live_drops)]
```

Comment:
Huh why is this needed, there are no owned values here

Reply:
I don't think it is needed, it's a leftover / copy-paste mistake.